### ISSUE TITLE
Suggestion: make tree collapsed at start

### ISSUE THREAD

**Issue by jstoebel** [pybee/cricket#63]
I'm not sure if there may be a compelling reason to not do this, but it might be useful to have the tree collapsed when cricket starts. If I am trying to navigate to a single suite or test, this would let me do less scrolling/collapsing.

**Comment by freakboy3742** [comment]
Agreed that something is needed here. My original inclination was a keyboard shortcut to enable easy expand all/contract all; but starting contracted might be another way.

Work on Cricket has been on hold while Toga was developing; however, we're now at a point where we can move forward, and we have a SoC student rebuilding Cricket in Toga. Hopefully we can address this fairly soon.

**Comment by freakboy3742** [comment]
I'm going to make a judgement call and close this WONTFIX. The Toga rewrite inadvertently implemented this behavior, and it was so annoying I added 'expand by default'.
